### PR TITLE
Update clickable header tutorial to use action button

### PR DESCRIPTION
**Pull request by keerlu**

### CODE REVIEW

File: howtos/how-to-make-a-clickable-avatar-in-a-header/README.md [hunk 1]
```diff
@@ -17,4 +17,11 @@ To run this tutorial, you will need:
 1. Clone or download the project
 2. Replace `<APP_ID>` in `server.js` with the value found in your [TalkJS dashboard](https://talkjs.com/dashboard/login).
 3. Run `npm install` to install dependencies.
-4. Run `npm start` to start the server.
\ No newline at end of file
+4. Run `npm start` to start the server.
+5. Add an action button to the chat header of your theme:
+   1. Go to the **Themes** tab of the TalkJS dashboard.
+   2. Select to **Edit** your current theme.
+   3. In the list of **Built-in Components**, select **ChatHeader**.
```
Comment (stevenwaterman) [inline howtos/how-to-make-a-clickable-avatar-in-a-header/README.md:24]: ChatHeader isn't available on all themes. Maybe worth adding a disclaimer? Specifically the group_chat is missing it. Or bringing it up with Frederik. He was last looking at this and concluded that it wasn't meant to have a chatheader component, but i'm not 100% sure what's going on with it tbh.

File: howtos/how-to-make-a-clickable-avatar-in-a-header/README.md [hunk 2]
```diff
@@ -17,4 +17,11 @@ To run this tutorial, you will need:
...
+   4. Find the code for displaying the user's name in the header (something like `<span>{{user.name}}</span>`) and replace it with the following:
+      `<span><ActionButton action="goToProfile" data-userId="{{user.id}}">{{user.name}}</ActionButton></span>`
```
Comment (stevenwaterman) [inline howtos/how-to-make-a-clickable-avatar-in-a-header/README.md:26]: Do we not want to make the avatar clickable too?

**Review by stevenwaterman — APPROVED** [review]

File: howtos/how-to-make-a-clickable-avatar-in-a-header/README.md (hunk 2 above)
Comment (keerlu) [reply howtos/how-to-make-a-clickable-avatar-in-a-header/README.md:26]: I was thinking that putting buttons on the names themselves would work better for group conversations (without a conversation name), so they could each have their own profile link. 

I could put a link on the avatar too for just the case where there's one other participant - will ask about that after publishing the current version

File: howtos/how-to-make-a-clickable-avatar-in-a-header/README.md (hunk 1 above)
Comment (keerlu) [reply howtos/how-to-make-a-clickable-avatar-in-a-header/README.md:24]: Thanks, added a disclaimer for now and will see what Frederik says

**Pull request merged**